python/kernmlops/kernmlops_benchmark/memcached.py

```diff
@@ -0,0 +1,104 @@
+import subprocess
+from dataclasses import dataclass
+from typing import cast
+
+from data_schema import GraphEngine, demote
+from kernmlops_benchmark.benchmark import Benchmark, GenericBenchmarkConfig
+from kernmlops_benchmark.errors import (
+  BenchmarkNotInCollectionData,
+  BenchmarkNotRunningError,
+  BenchmarkRunningError,
+)
+from kernmlops_config import ConfigBase
+
+
+@dataclass(frozen=True)
+class MemcachedConfig(ConfigBase):
+  operation_count: int = 1000000
+  read_proportion: float = 0.99
+  update_proportion: float = 0.01
+
+
+class MemcachedBenchmark(Benchmark):
```

Comment:
really this benchmark should be merged with `mongodb` and have it called `YCSBBenchmark` and have it take `database` as a config option to point at either `mongodb` or `memcached`

I am not sure if we should do that on this pr though -- but it would be nice

Reply:
let's do this now, this pr isn't rushed it's about adding to our benchmark base.

Reply:
Sounds good, we don't have a ton of other YCSB benchmarks to add right now though, so not sure if it needs to scale/be more confusing.